orcid-web/src/main/resources/freemarker/includes/print_record.ftl

```diff
@@ -18,7 +18,8 @@
 -->
 <#if RequestParameters['print']??>
 	<div class="print-orcid-record">
-		<a href="${baseUriHttp}/${(effectiveUserOrcid)!}" onClick="window.print();return false"><span class="glyphicon glyphicon-print"></span> Print your ORCID record</a>
+		<!--using hard-coded based url temporarily for local testing - need to replace with var for https version of base URL-->
+		<a id="printRecord" onclick="printPublicRecord('https://localhost:8443/orcid-web/${(effectiveUserOrcid)!}')"><span class="glyphicon glyphicon-print"></span> Print your ORCID record</a>		
```

Comment:
@mjcalvo21 Hard coded for localhost over https; not sure if we have a var available for https base uri...may need to create one.